sbt-dump-event-schema/src/main/scala/org/virtuslab/akkasaferserializer/DumpEventSchemaKeys.scala

```diff
@@ -0,0 +1,13 @@
+package org.virtuslab.akkasaferserializer
+
+import sbt.librarymanagement.ModuleID
+import sbt.{File, settingKey, taskKey}
+
+trait DumpEventSchemaKeys {
+  lazy val dumpEventSchema = taskKey[Unit]("Dump schema")
+  lazy val dumpEventSchemaOutputFile = taskKey[File]("Output file to dump schema to")
+  lazy val dumpEventSchemaOutputFilename = settingKey[String]("Filename to dump schema to")
+  lazy val dumpEventSchemaPlugin = settingKey[ModuleID]("ModuleId of dump schema plugin")
```

Comment:
Hmmm `...CompilerPlugin` here and below? 🤔 or does it refer to sbt plugin here (I don't think so)?